integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -162,7 +162,7 @@ iree_e2e_test_suite(
 )
 
 iree_vision_test_suite(
-    name = "vision_internal_tests",
+    name = "short_cifar10_internal_tests",
```

Comment:
If these are short, could you set the timeout to short. This would also implied by setting `test_size="small"` although I think these are probably at least "medium" tests https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes-tests. Yeah looking at the timeouts, I think this should be marked with `test_size="medium"`. Since "short" has a specific meaning in Bazel, calling this suite "short" is a bit confusing. How about "medium" instead? Or do the tests need more than 100MB of RAM?

```suggestion
    name = "medium_cifar10_internal_tests",
    test_size = "medium",
```

Reply:
Good catch! I didn't love the names and the fit much better :)